app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/ImageViewTouchFragment.kt

```diff
@@ -0,0 +1,57 @@
+package de.tum.`in`.tumcampusapp.component.other.generic
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.squareup.picasso.Callback
+import com.squareup.picasso.Picasso
+import de.tum.`in`.tumcampusapp.R
+import de.tum.`in`.tumcampusapp.utils.Utils
+
+class ImageViewTouchFragment : Fragment() {
+
+    lateinit var mRootView: View
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        mRootView = inflater.inflate(R.layout.fragment_image_view_touch, container, false)
+        return mRootView
+    }
+
+    fun loadImage(url: String, listener: ImageLoadingListener) {
+        val imageView = mRootView.findViewById<ImageView>(R.id.image_view_touch_fragment)
+        Utils.log("room finder url: $url")
+
+        val icon = ContextCompat.getDrawable(requireContext(), R.drawable.ic_outline_map_24px)
```

Comment:
Is the drawable used somewhere else? If not, we can consider adding the tint in the xml directly. This would remove the need for these two lines of code.

Reply:
Yes, it's used multiple times but I'm not sure if there is another one that uses the original color since the room finder doesn't work right now